prowler/providers/gcp/services/cloudstorage/cloudstorage_bucket_log_retention_policy_lock/cloudstorage_bucket_log_retention_policy_lock.metadata.json

```
@@ -1,26 +1,29 @@
 {
   "Provider": "gcp",
   "CheckID": "cloudstorage_bucket_log_retention_policy_lock",
-  "CheckTitle": "Ensure That Retention Policies on Cloud Storage Buckets Used for Exporting Logs Are Configured Using Bucket Lock",
+  "CheckTitle": "Cloud Storage log buckets have a Retention Policy with Bucket Lock enabled",
```

```suggestion
  "CheckTitle": "Cloud Storage log bucket has a Retention Policy with Bucket Lock enabled",
```
Based on [this](https://docs.prowler.com/developer-guide/check-metadata-guidelines#writing-guidelines).